extension/src/Template/TemplateFunctions.ts

```diff
@@ -135,3 +143,9 @@ function createXlfFiles(
     }
   }
 }
+function renumberObjects(appManifestPaths: string[]): void {
+  for (const appManifestPath of appManifestPaths) {
+    const folderPath = path.dirname(appManifestPath);
+    RenumberObjects.renumberObjectsInFolder(folderPath);
+  }
```

Comment:
Why not move this to RenumberObjects.ts as well? :)

Reply:
Just since it will not be used from anywhere else then I'm this file...